native/acter/src/api/auth.rs

```diff
@@ -332,3 +342,19 @@ pub async fn register_with_token_under_config(
         })
         .await?
 }
+
+impl Client {
+    pub async fn deactivate(&self, password: String) -> Result<bool> {
+        // ToDo: make this a proper User-Interactive Flow rather than hardcoded for
+        //       password-only instance.
+        let account = self.account()?;
+        RUNTIME
+            .spawn(async move {
+                let auth_data =
+                    AuthData::Password(Password::new(account.user_id().into(), password.clone()));
+                account.deactivate(None, Some(auth_data)).await?;
```

Comment:
I think it is better to deactivate after logout.
Then `deactivate()` would be outside of `Client` crate.

Reply:
We can't deactivate after we are logged out. It requires a logged-in client to do.